cross-chain/arbitrum/deploy_l2/12_update_wormhole_gateway_mapping.ts

```diff
@@ -0,0 +1,29 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, ethers } = hre
+  const { execute } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // See https://book.wormhole.com/reference/contracts.html
+  // This ID is valid for both Arbitrum Goerli and Mainnet
+  const wormholeChainID = 23
+
+  const ArbitrumWormholeGateway = await deployments.get(
+    "ArbitrumWormholeGateway"
+  )
+
+  await execute(
+    "ArbitrumWormholeGateway",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "updateGatewayAddress",
+    wormholeChainID,
+    ethers.utils.hexZeroPad(ArbitrumWormholeGateway.address, 32)
+  )
+}
+
+export default func
+
+func.tags = ["GatewayAddress"]
```

Comment:
We usually start a tag name with a verb, so maybe `SetGatewayAddress`?

Reply:
[33a2ace](https://github.com/keep-network/tbtc-v2/pull/554/commits/33a2acea2e720e32a2d125f1cdba2f828997bf39)